osdk-core/modules/src/filemgr/impl/dji_file_mgr_impl.cpp

```diff
@@ -139,7 +139,7 @@ void FileMgrImpl::fileListMonitorTask(void *arg) {
     uint32_t curTimeMs = 0;
     uint32_t preTimeMs = 0;
     uint32_t pollTimeMsInterval = 500;
-    uint32_t taskTimeOutMs = 6000;
+    uint32_t taskTimeOutMs = 6000 * 5;
```

Comment:
This will extend to 30 seconds, is it help to improve the stability?

Reply:
@RobertLiu-Atnm , yes. 

Time between datapacks in OnReceiveDataPack would increase as more-and-more file data slices where acquired/requested.

Reply:
occasionally it would take "onFirstPackReceived" after calling `startReqFileListbySlices`for subsequent slices.

Reply:
30 seconds might be overkill, but it seems like this number occasionally is increased anyway, so with 30 seconds it gives us additional space.